src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -128,27 +146,81 @@ To learn more, refer to [Deprovision a user](https://help.okta.com/en-us/Content
 
 A linked group that is unassigned from the app integration continues to appear in {{ site.data.products.db }} unless it is unlinked. Refer to [Automate Group Management](#automate-group-management).
 
-### Automate group management
+<a id="automate-group-management"></a>
+### Automate group management with Okta
 
-By default, users and groups are provisioned in {{ site.data.products.db }} only when they are assigned to the app integration, and changes to a group's membership in Okta are not automatically reflected in {{ site.data.products.db }}. When [Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-about-group-push.htm) is enabled in an app integration, modifying group membership in Okta is automatically reflected in {{ site.data.products.db }}.
+The following sections show how to enable and manage Okta's Group Push feature. For more information and troubleshooting, refer to [Manage Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-group-push-main.htm) in the Okta documentation.
 
-To enable Group Push and link groups:
+### Enable Group Push
```

Comment:
This is part of the above Automate group management section, right? If yes, Should this have a #### header instead of ###?

Reply:
Did I misunderstand your feedback in https://github.com/cockroachdb/docs/pull/17278#discussion_r1240896501? 
![Screenshot 2023-06-29 at 1 27 25 PM](https://github.com/cockroachdb/docs/assets/7674613/8f586703-02de-45c6-87ec-dfdef36d6380)